src/detect-parse.h

```diff
@@ -106,6 +89,13 @@ int WARN_UNUSED DetectSignatureSetMultiAppProto(Signature *s, const AppProto *al
 
 /* parse regex setup and free util funcs */
 
+#ifndef SURICATA_BINDGEN_H
```

Comment:
would it make sense to break this out into its own file?

Reply:
if so, can be done as a follow up post-merge

Reply:
> would it make sense to break this out into its own file?

It can make sense.

But more generally, we want to get rid of these, right ? (Does it have a ticket ? )